distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -70,6 +70,10 @@ if [ -f "$DEFAULT" ]; then
 	. "$DEFAULT"
 fi
 
+if [ "$ES_USER" != "elasticsearch" ] || [ "$ES_GROUP" != "elasticsearch" ]; then
```

Comment:
Right above these we source `$DEFAULT` (which is /etc/default/elasticsearch), so someone can change /etc/default/elasticsearch after installation to export `ES_USER` and/or `ES_GROUP` which would then be used in this file, so I needed to place this after the sourcing